developer/src/kmc-keyboard/test/fixtures/sections/keys/maximal.xml

```diff
@@ -24,7 +24,7 @@
       <row keys="Q q W w" />
     </layer>
   </layers>
-  <layers form="hardware" hardware="abnt2">
+  <layers form="iso">
```

Comment:
```suggestion
  <layers form="abnt2">
```
?

Reply:
abnt2 not supported, so changed to iso since this is now being validated